crates/services/consensus_module/poa/src/signer.rs

```diff
@@ -68,6 +69,30 @@ impl SignMode {
         };
         Ok(Consensus::PoA(PoAConsensus::new(poa_signature)))
     }
+
+    pub fn public_key_address(&self) -> anyhow::Result<Option<Bytes32>> {
+        match self {
+            SignMode::Unavailable => Ok(None),
+            SignMode::Key(secret_key) => {
+                Ok(secret_key.expose_secret().public_key().hash().into())
```

Comment:
assuming that what needs to be logged is the public_key_hash corresponding to the secret key. Is this correct?

Reply:
Closing: this assumption does not hold anymore as I am now using the `Input::owner()` function.